src/WpfMath/Data/DefaultTexFont.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version='1.0'?>
 <!--
-  DefaultTeXFont.xml
+  DefaultTexFont.xml
   =========================================================================
-  This file is part of the JMathTeX Library - http://jmathtex.sourceforge.net
+  This file is originally a part of the JMathTeX Library - http://jmathtex.sourceforge.net

```

Comment:
I have originally written it like that, but have copy-pasted the wording from [JLaTeXMath](https://github.com/opencollab/jlatexmath/blob/7b99ab584bd8598194d35f60965bda9eac92d641/src/org/scilab/forge/jlatexmath/GlueSettings.xml#L5). I'll fix that :)